tradewave/views.py

```diff
@@ -1090,21 +1103,112 @@ def redeem_credits(request):
         return redirect('tradewave:login', status_msg=e.message)
 
 
-# *** handler for creating a new user ***
+# *** view handler for creating a new user ***
```

Comment:
@dannypernik Since each page can be viewed through multiple pathways through the system, it can quickly get overwhelming to keep track of all the possible contexts (inputs). This is a descriptive header format I came up with to make it this task much easier.